# HG changeset patch
# User Rik <rik@octave.org>
# Date 1407286307 25200
#      Tue Aug 05 17:51:47 2014 -0700
# Branch gui-release
# Node ID 0b657f4e7780bfeffc65895c5dad7fe73aa37e8b
# Parent  724fc0a9eeb1c8f67190ee98fd78b3a4b04830b5
# Parent  082063dc2165c71bb25457c7cb7db81fdb068595
maint: Merge stable to gui-release.

diff --git a/doc/interpreter/package.txi b/doc/interpreter/package.txi
--- a/doc/interpreter/package.txi
+++ b/doc/interpreter/package.txi
@@ -266,17 +266,17 @@ the package functions were installed.
 @item package/on_uninstall.m
 This is an optional function that is run prior to the removal of a
 package.  This function is called with a single argument, a struct with
 fields names after the data in the @file{DESCRIPTION}, the paths where
 the package functions are installed, and whether the package is currently
 loaded.
 @end table
 
-Besides the above mentioned files, a package can also contain on or
+Besides the above mentioned files, a package can also contain one or
 more of the following directories:
 
 @table @code
 @item package/inst
 An optional directory containing any files that are directly installed
 by the package.  Typically this will include any @code{m}-files.
 
 @item package/src
@@ -354,17 +354,17 @@ License: GPLv3+
 The package manager currently recognizes the following keywords
 
 @table @code
 @item Name
 Name of the package.
 
 @item Version
 Version of the package.  A package version must be 3 numbers separated
-by a dot.
+by dots.
 
 @item Date
 Date of last update.
 
 @item Author
 Original author of the package.
 
 @item Maintainer
diff --git a/scripts/miscellaneous/ls.m b/scripts/miscellaneous/ls.m
--- a/scripts/miscellaneous/ls.m
+++ b/scripts/miscellaneous/ls.m
@@ -63,17 +63,17 @@ function retval = ls (varargin)
     if (ispc () && ! isunix ())
       ## shell (cmd.exe) on MinGW uses '^' as escape character
       args = regexprep (args, '([^\w.*? -])', '^$1');
       ## Strip UNIX directory character which looks like an option to dir cmd.
       if (args{end}(end) == '/')
         args{end}(end) = "";
       endif
     else
-      args = regexprep (args, '([^\w.*? -])', '\\$1');
+      args = regexprep (args, '([^\w.*?-])', '\\$1');
     endif
     args = sprintf ("%s ", args{:});
   else
     args = "";
   endif
 
   cmd = sprintf ("%s %s", __ls_command__, args);
 
diff --git a/scripts/special-matrix/toeplitz.m b/scripts/special-matrix/toeplitz.m
--- a/scripts/special-matrix/toeplitz.m
+++ b/scripts/special-matrix/toeplitz.m
@@ -69,17 +69,17 @@ function retval = toeplitz (c, r)
 
     r = c;
     nr = length (c);
     nc = nr;
   else
     if (! (isvector (c) && isvector (r))) 
       error ("toeplitz: C and R must be vectors");
     elseif (r(1) != c(1))
-      warning ("toeplitz: column wins anti-diagonal conflict");
+      warning ("toeplitz: column wins diagonal conflict");
     endif
 
     nr = length (c);
     nc = length (r);
   endif
 
   if (nr == 0 || nc == 0)
     ## Empty matrix.
